aws-sns-topic/src/main/java/software/amazon/sns/topic/BaseHandlerStd.java

```diff
@@ -1,6 +1,8 @@
 package software.amazon.sns.topic;
 
 import com.google.common.collect.Sets;
+import software.amazon.awssdk.awscore.AwsRequest;
+import software.amazon.awssdk.awscore.AwsResponse;
```

Comment:
I don't see these two import used below. Should we remove them?

Reply:
Oops, that's for stabilize which I didn't make it work. I'll remove them in the next fix. This one is kinda urgent.

Reply:
Sounds good